### ISSUE TITLE
WHAT IS YOUR FAVORITE SEA CREATURE?!

### ISSUE THREAD

**Issue by iuloshi** [khandelwal/AskShashank#5]

**Comment by theresaanna** [comment]
YES PLEASE! Some suggestions, from the original thread: :whale: :turtle: :dolphin: :fish: 
Also :tropical_fish: :whale2: :octopus: :water_buffalo:

**Comment by theresaanna** [comment]
Also :turtle: is the right answer.

**Comment by jehlers** [comment]
of the emoji variety, I vote: :tropical_fish: or :octopus:

also does a :snail: count?
